frontend/src/helpers/exam.js

```diff
@@ -1,4 +1,5 @@
 import moment from "moment";
+import axios from 'axios';
```

Comment:
not sure you need that

Reply:
Thought I erased it in the last commit thx